### ISSUE TITLE
Homebrew install MacOS 10.13 fails without outdated and incompatible python modules

### ISSUE THREAD

**Issue by mvilain** [oracle/oci-cli#486]
The current brew package for oci-cli fails to install on my 10.13.6 system. It gets to installing the cryptogrphy python module, then barfs.

```
==> /usr/local/Cellar/oci-cli/3.3.1/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--cryptography-202
Last 15 lines from /Users/vilain/Library/Logs/Homebrew/oci-cli/07.pip:
  WARNING: Discarding https://files.pythonhosted.org/packages/75/50/973397c5ba854445bcc396b593b5db1958da6ab8d665b27397daa1497018/tomli-1.2.1.tar.gz#sha256=a5b75cb6f3968abb47af1b40c1819dc519ea82bcc065776a866e8d74c5ca9442 (from https://pypi.org/simple/tomli/) (requires-python:>=3.6). Command errored out with exit status 2: /usr/local/Cellar/oci-cli/3.3.1/libexec/bin/python3.10 /private/tmp/pip-standalone-pip-at300nl9/__env_pip__.zip/pip install --ignore-installed --no-user --prefix /private/tmp/pip-build-env-53xkjoj0/overlay --no-warn-script-location --no-binary :all: --only-binary :none: -i https://pypi.org/simple -- 'flit_core>=3.2.0,<4' Check the logs for full command output.
  Collecting setuptools-rust>=0.11.4
    Downloading setuptools-rust-1.1.0.tar.gz (284 kB)
    Installing build dependencies: started
    Installing build dependencies: finished with status 'done'
    Getting requirements to build wheel: started
    Getting requirements to build wheel: finished with status 'done'
  ERROR: Some build dependencies for setuptools-rust>=0.11.4 from https://files.pythonhosted.org/packages/a1/00/56ed41e98bd83e8fd028e24144e20bef87cdec0110fd1729352fce33d36c/setuptools-rust-1.1.0.tar.gz#sha256=e8a65f7398028cb0e4e00730ee3d1e168bf9d4e3770a61fec815e8e7a24dcab1 conflict with the backend dependencies: setuptools_scm==6.0.1 is incompatible with setuptools_scm>=6.3.2.
  WARNING: You are using pip version 21.2.4; however, version 21.3.1 is available.
  You should consider upgrading via the '/usr/local/Cellar/oci-cli/3.3.1/libexec/bin/python3.10 -m pip install --upgrade pip' command.
  Installing build dependencies: finished with status 'error'
WARNING: Discarding file:///private/tmp/oci-cli--cryptography-20211202-27791-1ovbc6v/cryptography-3.4.7. Command errored out with exit status 1: /usr/local/Cellar/oci-cli/3.3.1/libexec/bin/python3.10 /private/tmp/pip-standalone-pip-at300nl9/__env_pip__.zip/pip install --ignore-installed --no-user --prefix /private/tmp/pip-build-env-ld_2atud/overlay --no-warn-script-location --no-binary :all: --only-binary :none: -i https://pypi.org/simple -- 'setuptools>=40.6.0' wheel 'cffi>=1.12; platform_python_implementation != '"'"'PyPy'"'"'' 'setuptools-rust>=0.11.4' Check the logs for full command output.
ERROR: Command errored out with exit status 1: /usr/local/Cellar/oci-cli/3.3.1/libexec/bin/python3.10 /private/tmp/pip-standalone-pip-at300nl9/__env_pip__.zip/pip install --ignore-installed --no-user --prefix /private/tmp/pip-build-env-ld_2atud/overlay --no-warn-script-location --no-binary :all: --only-binary :none: -i https://pypi.org/simple -- 'setuptools>=40.6.0' wheel 'cffi>=1.12; platform_python_implementation != '"'"'PyPy'"'"'' 'setuptools-rust>=0.11.4' Check the logs for full command output.
WARNING: You are using pip version 21.2.4; however, version 21.3.1 is available.
You should consider upgrading via the '/usr/local/Cellar/oci-cli/3.3.1/libexec/bin/python3.10 -m pip install --upgrade pip' command.

Do not report this issue to Homebrew/brew or Homebrew/core!
```

**Comment by mvilain** [comment]
Problem fixed on 3.4 which was updated on Dec 14. It's now working.
`
(venv) 09:05:01 [dexlabs 146 ~/gitlab/aws]$ brew info oci-cli
oci-cli: stable 3.4.0, HEAD
Oracle Cloud Infrastructure CLI
https://docs.cloud.oracle.com/iaas/Content/API/Concepts/cliconcepts.htm
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/oci-cli.rb
License: UPL-1.0 or Apache-2.0
==> Dependencies
Build: rust ✔
Required: python@3.10 ✔, six ✔
==> Options
--HEAD
	Install HEAD version
==> Analytics
install: 2,365 (30 days), 7,535 (90 days), 22,555 (365 days)
install-on-request: 2,363 (30 days), 7,530 (90 days), 22,547 (365 days)
build-error: 16 (30 days)
(venv) 09:05:13 [dexlabs 147 ~/gitlab/aws]$ brew install oci-cli
Running `brew update --preinstall`...
==> Auto-updated Homebrew!
Updated 3 taps (hashicorp/tap, homebrew/core and homebrew/cask).
==> Updated Formulae
Updated 17 formulae.
==> Updated Casks
Updated 73 casks.

Warning: You are using macOS 10.13.
We (and Apple) do not provide support for this old version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels. You are responsible for resolving
any issues you experience while you are running this
old version.

==> Downloading https://files.pythonhosted.org/packages/25/e2/85d4a709a3ea58f8e36b4db9eb7927560a2fa4b6f8f362fb6475962fec51/arrow-1.2.1.tar.gz
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/6c/ae/d26450834f0acc9e3d1f74508da6df1551ceab6c2ce0766a593362d6d57f/certifi-2021.10.8.tar.gz
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/00/9e/92de7e1217ccc3d5f352ba21e52398372525765b2e0c4530e6eb2ba9282a/cffi-1.15.0.tar.gz
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/27/3f/2e8de6f352fddfd07eca3072b4757f68f5dd2032d4b2ad8e115c6b1bf5bc/circuitbreaker-1.3.2.tar.
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/27/6f/be940c8b1f1d69daceeb0032fee6c34d7bd70e3e649ccac0951500b4720e/click-7.1.2.tar.gz
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/9b/77/461087a514d2e8ece1c975d8216bc03f7048e6090c5166bc34115afdaa53/cryptography-3.4.7.tar.gz
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/3c/56/3f325b1eef9791759784aa5046a8f6a1aff8f7c898a2e34506771d3b99d8/jmespath-0.10.0.tar.gz
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/16/56/ffeb1f64dcb640eafee19e7baa5244248198d4c88081182498f718077438/oci-2.53.0.tar.gz
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/5e/0b/95d387f5f4433cb0f53ff7ad859bd2c6051051cebbb564f139a999ab46de/pycparser-2.21.tar.gz
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/0d/1d/6cc4bd4e79f78be6640fab268555a11af48474fac9df187c3361a1d1d2f0/pyOpenSSL-19.1.0.tar.gz
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/4c/c4/13b4776ea2d76c115c1d1b84579f3764ee6d57204f6be27119f13a61d0a9/python-dateutil-2.8.2.tar
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/e3/8e/1cde9d002f48a940b9d9d38820aaf444b229450c0854bdf15305ce4a3d1a/pytz-2021.3.tar.gz
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/a0/a4/d63f2d7597e1a4b55aa3b4d6c5b029991d3b824b5bd331af8d4ab1ed687d/PyYAML-5.4.1.tar.gz
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/9b/c4/4a21174f32f8a7e1104798c445dacdc1d4df86f2f26722767034e4de4bff/terminaltables-3.1.0.tar.
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/7c/9c/79ceb6b086ed4994f173adb627c81d05257cf21790617f37c7be8988e0d9/oci-cli-3.4.0.tar.gz
######################################################################## 100.0%
==> python3.10 -m venv --system-site-packages /usr/local/Cellar/oci-cli/3.4.0/libexec
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--arrow-20211217-6
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--certifi-20211217
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--cffi-20211217-63
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--circuitbreaker-2
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--click-20211217-6
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--cryptography-202
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--jmespath-2021121
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--oci-20211217-636
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--pycparser-202112
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--pyOpenSSL-202112
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--python-dateutil-
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--pytz-20211217-63
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--PyYAML-20211217-
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli--terminaltables-2
==> /usr/local/Cellar/oci-cli/3.4.0/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/oci-cli-20211217-63638-1q
🍺  /usr/local/Cellar/oci-cli/3.4.0: 25,686 files, 256.3MB, built in 10 minutes 46 seconds
==> Running `brew cleanup oci-cli`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
`

**Comment by mvilain** [comment]
fixed in 3.4

[oci-install-3.4.txt](https://github.com/oracle/oci-cli/files/7736780/oci-install-3.4.txt)
